docs/source/video_tutorials.rst

```diff
@@ -0,0 +1,39 @@
+Video Tutorials
```

Comment:
Can I have my name somewhere on this page?

Reply:
For sure, but I was hoping you would add it. But I guess something like "Below is a list of video tutorials made by Andrew Tarzia, which provide a thorough introduction to stk."

Reply:
Perfect! Link to my github with my name?

Reply:
:+1:  Sounds good to me